test/functional/unit-sdam/srv_polling.test.js

```diff
@@ -299,7 +299,7 @@ describe('Mongos SRV Polling', function () {
         return records.map(r => `${r.name}:${r.port}`);
       }
 
-      const MONGOS_DEFAULT_ISMASTER = Object.assign({}, mock.DEFAULT_ISMASTER_36, {
+      const MONGOS_LEGACY_HELLO = Object.assign({}, mock.HELLO, {
```

Comment:
Is it really LEGACY_HELLO if we are using the most up to date HELLO? Perhaps `MONGOS_HELLO` would work here?

Reply:
Updated to be `MONGOS_HELLO` now.